packages/core/src/kernelCoreTemplate.js

```diff
@@ -82,6 +85,31 @@
     Object.freeze(kernel)
     return kernel
 
+    function scuttleGlobalThis() {
+      let props = [...Object.getOwnPropertyNames(globalThis)]
+      if (globalThis?.Window?.prototype) {
+        props = [...props, ...Object.getOwnPropertyNames(Window.prototype)]
+      }
+      if (globalThis?.EventTarget?.prototype) {
+        props = [...props, ...Object.getOwnPropertyNames(EventTarget.prototype)]
+      }
+
+      const avoids = [
+        // non configurables
+        'location', 'top', 'global', 'self', 'window', 'Infinity', 'NaN', 'document',
+        // support LM,SES exported APIs
+        'LavaPack', 'Compartment',
+        // support polyfill
+        'globalThis',
+      ]
+
+      for (const prop of props) {
+        if (!avoids.includes(prop)) {
+          Object.defineProperty(window, prop, {value: undefined})
```

Comment:
How about a getter that throws with helpful information?

Reply:
fd49fac76

Reply:
also, for scuttling to work, lockdown umd must access the new scoped version of Error, instead of directly via globalThis, so had to fix that too 760188307a17be81c3673049e5f740a9aee32b02
is this how we update lockdown umd? should it be updated also elsewhere?

Reply:
ok